scripts/verify_env.py

```diff
@@ -0,0 +1,44 @@
+#!/usr/bin/env python
+"""
+Used to check missing deployed lambda env variables with local environment variables.
+Does not check
+
+"""
+import os
+import boto3
+import json
+
+
+ssm_client = boto3.client("ssm")
+stages = ['dev', 'integration', 'staging', 'prod']
```

Comment:
What does this variable do?

Reply:
it was hitchhiking, just got on a train of of town..... *removed*